assets/models/system/Mistral-7B-Instruct-v0-1/spec.yaml

```diff
@@ -0,0 +1,35 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: Mistral-7B-Instruct-v0-1
+path: ./
+properties:
+  SHA: 3dc28cf29d2edd31a0a7b8f0b21637059815b4d5
+  inference-recommended-sku: Standard_E16s_v3,Standard_NC12s_v3, Standard_NC24s_v3, Standard_NC24rs_v3,
+    Standard_ND40rs_v2, Standard_ND96asr_v4, Standard_ND96amsr_A100_v4
+  evaluation-recommended-sku: Standard_NC6s_v3,Standard_NC12s_v3, Standard_NC24s_v3, Standard_NC24rs_v3,
+    Standard_ND40rs_v2, Standard_ND96asr_v4, Standard_ND96amsr_A100_v4
+  languages: EN
+  azureml.copyRegistryFilesToWorkspace: true
```

Comment:
I don't think this is necessary for this model?

Reply:
correct. Removed